server/sonar-web/src/main/js/apps/coding-rules/templates/coding-rules-workspace-list-item.hbs

```diff
@@ -17,6 +17,8 @@
     <td>
       <div class="coding-rule-title">
         <a class="js-rule link-no-underline" href="{{permalink}}">{{name}}</a>
+        <span class="outline-badge spacer-left" title="{{t 'coding_rules.rule_template.title'}}" 
```

Comment:
I think you forgot a check on `isTemplate`, currently all rules have the badge

Reply:
🙀